cg/meta/workflow/rnafusion.py

```diff
@@ -63,18 +63,8 @@ def process(self):
         return self._process
 
     @process.setter
-    def process(self, use_nextflow: bool):
-        if use_nextflow:
-            self._process = Process(
-                binary=self.config.rnafusion.binary_path,
-                environment=self.conda_env,
-                conda_binary=self.conda_binary,
-                launch_directory=self.config.rnafusion.launch_directory,
-            )
-        else:
-            self._process = Process(
-                binary=self.tower_binary_path,
```

Comment:
Did you miss defining the tower process as the default one in the init?  Or where is it defined now?

Reply:
It's here: 
https://github.com/Clinical-Genomics/cg/blob/30f68d9666add4dbb9660c772270bed96450e0d2/cg/meta/workflow/rnafusion.py#L57-L63

Reply:
Sorry, missed it, thanks <3